docs/operator-manual/config-management-plugins.md

```diff
@@ -354,6 +354,16 @@ You can set it one of three ways:
 2. The `reposerver.plugin.tar.exclusions` key if you are using `argocd-cmd-params-cm`
 3. Directly setting `ARGOCD_REPO_SERVER_PLUGIN_TAR_EXCLUSIONS` environment variable on the repo server.
 
+In addition, if you're using a monorepo mixed with code and GitOps resources, you probably want to only include files 
```

Comment:
What's the behavior if I specify both inclusions and exclusions?

Reply:
You're right, it seems redundant but the main advantage of using `exclusions` is that it will speed up the files scan because of `filepath.SkipDir`.

Here is  what we have in our config for instance:

`  
reposerver.plugin.tar.inclusions: >-
    **/deploy/**
reposerver.plugin.tar.exclusions: >-
    .git/**
`

All the files living in a deploy folder (or sub-folder) will be passed to the sidecar and all git files will be exluded.